README.md

```diff
@@ -1,28 +1,11 @@
-# Exonum Java Binding
-
-[![Build Status](https://img.shields.io/travis/exonum/exonum-java-binding/master.svg)](https://travis-ci.com/exonum/exonum-java-binding)
-[![Maven Central](https://img.shields.io/maven-central/v/com.exonum.binding/exonum-java-binding-core.svg)](https://mvnrepository.com/artifact/com.exonum.binding/exonum-java-binding-core/latest)
-[![Coverage Status](https://coveralls.io/repos/github/exonum/exonum-java-binding/badge.svg?branch=master)](https://coveralls.io/github/exonum/exonum-java-binding?branch=master)
-[![Join the chat on https://gitter.im/exonum/exonum-java-binding](https://img.shields.io/gitter/room/exonum/exonum-java-binding.svg?label=Chat)](https://gitter.im/exonum/exonum-java-binding)
-
-Exonum Java Binding is a framework for building blockchain applications in Java, 
-powered by Exonum.
-
-The project is at alpha stage. Currently you can define a stateful Exonum Service 
-in Java, configure a network of nodes and run the service in it.
-
-For more information see the relevant documentation on the tool:
-  - [What is Exonum?](https://exonum.com/doc/version/latest/get-started/what-is-exonum/)
-  - [User Guide](https://exonum.com/doc/version/latest/get-started/java-binding/)
-  - [Cryptocurrency Example Service](exonum-java-binding-cryptocurrency-demo)
-  - [Roadmap](ROADMAP.md)
-  - [Contribution Guide](CONTRIBUTING.md)
-  - [Code of Conduct](https://github.com/exonum/exonum/blob/master/CODE_OF_CONDUCT.md)
-
-If you are using Exonum in your project and want to be listed on our website and
-GitHub list — write us a line to <exonum@bitfury.com>.
-
-
+# Exonum Library for Java
+This is a primary repository for Exonum projects for java.  
```

Comment:
```suggestion
This is a primary repository for Exonum projects in Java.  
```

Though this is not true. i.e. this repository will not be used for our Java services.